tests/e2e/tools/lib/covers.js

```diff
@@ -53,34 +52,11 @@ async function addCoversBlock(page, editor, style = '') {
     await tagsInput.type(TAG_NAMES[3]);
     await suggestions.getByRole('option').first().click();
   }
-
-  // Default style, i.e 'Content cover'.
-  if (style === 'Content') {
-    // Fill in the Post types.
-    const postTypesInput = await page.getByLabel('Select Post Types');
-    await postTypesInput.scrollIntoViewIfNeeded();
-
-    await postTypesInput.type(POST_TYPES[0]);
-    await suggestions.getByRole('option').first().click();
-
-    await postTypesInput.type(POST_TYPES[1]);
-    await suggestions.getByRole('option').first().click();
-  }
   await page.getByLabel('Button Text').fill('Read more');
 }
 
 async function checkCoversBlock(page, style) {
-  // Default style, i.e 'Content cover'.
-  if (style === 'Content') {
-    await expect(page.locator('.content-covers-block')).toBeVisible();
-    const frontendCovers = await page.locator('.post-column.cover').all();
-    for (const [, cover] of frontendCovers.entries()) {
-      await expect(cover.locator('.content-covers-block-image > a > img')).toBeVisible();
-      await expect(cover.locator('.content-covers-block-information > h5 > a')).toBeVisible();
-      await expect(cover.locator('.content-covers-block-information > .publication-date')).toBeVisible();
-      await expect(cover.locator('.content-covers-block-information > .post-excerpt')).toBeVisible();
-    }
-  } else if (style === 'Campaign') {
+  if (style === 'Campaign') {
```

Comment:
Aren't we supposed to remove this part of the function too? Since we already retired the Campaign style?

Reply:
Done!